scripts/parse_sparse_output.sh

```diff
@@ -27,7 +27,10 @@ main()
 
     sparse_errors+=(-e '[[:space:]]error:[[:space:]]')
 
-    sparse_errors+=(-e '[[:space:]]warning:[[:space:]].*different address space')
+    # TODO: fix mtl source code
+    if test 'mtl' != "$platform"; then
+        sparse_errors+=(-e '[[:space:]]warning:[[:space:]].*different address space')
+    fi
 
     ! grep -v 'alsatplg.*topology2.*skip' | grep -i  "${sparse_errors[@]}"
 
```

Comment:
this means, we will see complete MTL sparse output including "address space" warnings in CI, it just won't be a failure? Could you maybe remove "SKIP CI" to let us see what it will look like?

Reply:
Indeed, thanks! I'm not sure whether it's a good idea to green-wash our failures, but as long as we have the output, I personally can live with it

Reply:
I don't like it either but it's been red for too long now. I also made sure it's very easy to revert.

Can you approve then?

Reply:
@lgirdwood re-submitting the other patches is not an issue but does anyone have any idea how to fix the current warnings on MTL and a line of sight on a fix? Right now we are actively training users to ignore the results on MTL and we are blind to possibly WORSE MTL regressions.

Reply:
Will be fixed after v2.4 is tagged, IOW early next week.